app/src/main/java/com/instructure/template/projectCodeHere/ProfilePage.java

```diff
@@ -0,0 +1,184 @@
+                e.printStackTrace();
+            }
+            ApiPrefs.clearAllData();
+            // Take us back to login starting point
+            Intent intent = new Intent(requireContext(), LoginActivity.class);
+            intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+            intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK);
+            startActivity(intent);
+        });
+        Retrofit client = (new Retrofit.Builder()).baseUrl(ApiPrefs.getFullDomain()).addConverterFactory(GsonConverterFactory.create()).build();
+        GetProfile getProfile = client.create(GetProfile.class);
+        Call<GetProfile.ProfileResponse> call = getProfile.profileCall(ApiPrefs.getUser().getId(),"Bearer " + ApiPrefs.getToken(), ApiPrefs.getUserAgent());
+        call.enqueue((new Callback<GetProfile.ProfileResponse>() {
+            public void onFailure(@NotNull Call call, @NotNull Throwable t) {
+                // This  is where you would put code for the error/failure case
+                Log.d("ERROR","call has failed");
+                Log.d("call",call.toString());
+                Log.d("throwable",t.toString());
+            }
+
+            public void onResponse(@NotNull Call call, @NotNull Response response) {
+                // This is where you would put code for the success case!
+                // The data is in the response body - response.body()
+                GetProfile.ProfileResponse g;
+                g = (GetProfile.ProfileResponse)response.body();
+                textView.setText(g.getName());
+                new DownLoadImageTask((ImageView) view.findViewById(R.id.profileIcon)).execute(g.getAvatar_url());
+                textView3.setText(g.getPrimary_email());
+            }
+        }));
+        GetEnrollments getEnrollments = client.create(GetEnrollments.class);
+        Call<List<GetEnrollments.EnrollmentResponse>> call2 = getEnrollments.enrollmentsCall(ApiPrefs.getUser().getId(), "Bearer " + ApiPrefs.getToken(), ApiPrefs.getUserAgent());
+        call2.enqueue((new Callback<List<GetEnrollments.EnrollmentResponse>>() {
+            public void onFailure(@NotNull Call call2, @NotNull Throwable t) {
+                // This  is where you would put code for the error/failure case
+            }
+
+            public void onResponse(@NotNull Call call, @NotNull Response response) {
+                // This is where you would put code for the success case!
+                // The data is in the response body - response.body()
+                List<GetEnrollments.EnrollmentResponse> g;
+                g = (ArrayList<GetEnrollments.EnrollmentResponse>)response.body();
+                int count=0;
+                double total=0;
+                for(GetEnrollments.EnrollmentResponse i : g) {
+                    total+=i.getGrades().getCurrent_score();
+                    count++;
+                }
+                double totalGPA=total/count,GPA=0;
+                if(totalGPA>90) {
```

Comment:
This sucks. Can't you implement something equivalent to `(score_in_percentage / 20) - 1` instead? What if someone has a 130% on something?